## Hi there ðŸ‘‹

- I am a **molecular biologist turned computational biologist** with expertise in **genomics, epigenomics, transcriptomics**, and **multi-omics data integration** â€” basically anything with way too many rows and columns.  
- I am an **inquisitive learner** and taught myself **R** from scratch during my PhD â€” starting with online courses and tutorials, learning to code, and compiling scripts that actually worked.  
- During my PhD, I analyzed **TCGA** and **ENCODE** datasets to study the impact and regulation of **imprinted microRNA clusters** in human cancers â€” using **predictive ML algorithms** to do the heavy lifting.  
- Currently, I analyze **bulk and single-cell RNA-seq** and **ATAC-seq** to uncover regulatory mechanisms in **non-classical food allergies**.  
- I have developed a knack for presenting large, complex datasets into **clean, intuitive visualizations** â€” my version of **runnerâ€™s high**.  
- I primarily use **R** (tidyverse, DESeq2, Seurat) for statistical analysis and visualization and **Python** when my workflows demand it, building reproducible **bioinformatics pipelines** using **Unix/Linux, HPC/SLURM and Nextflow**.  
- Currently juggling an **MS in Bioinformatics at NYU**, continuing my habit of learning new things. Long-term, I hope to merge **wet-lab intuition** with **computational wizardry** to exploreâ€”and clearly communicateâ€”the complexity of biological systems.  

**Fun fact:** When not in the lab, I lead **STEM & postdoc communities**, champion **women and international postdocs**, read **fiction** voraciously, and bring **main-character energy** to dance and stage performances. 
